index.html

```diff
@@ -2,283 +2,179 @@
+      <ul class="socials-main">
+        <li><a href="#"><img src="images/github.svg" alt="Github"></a></li>
+        <li><a href="#"><img src="images/Linkedin-icon.svg" alt="linkedin"></a></li>
+        <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+        <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+        <li><a href="#"><img src="images/medium.svg" alt="medium"> </a></li>
+      </ul>
     </section>
+  </div>
 
-    <!-- card 3 -->
-    <section class="work-container3">
-        <div class="portfolio-img desktop-img-2">
-            <img src="images/Snapshoot-Portfolio-3.png" alt="card-3-img">
-        </div>
-
-        <div class="work-content">
-
-
-            <h2 class="tonic0">Tonic</h2>
-
-            <ul class="prop">
-                <li class="can">CANOPY</li>
-                <li class="bac"><img src="images/Counter.png" alt="dot"> Back End Dev</li>
-                <li class="num"><img src="images/Counter.png" alt="dot"> 2015</li>
-            </ul>
-
-            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
-                required.</p>
-
-            <ul class="stack">
-                <li class="html">html</li>
-                <li class="ruby">Ruby on rails</li>
-                <li class="css">css</li>
-                <li class="js">javaScript</li>
-            </ul>
-
-            <div class="button">
-                <button type="button">See Project</button>
-            </div>
-        </div>
-    </section>
+  <div class="portfolio-container" id = "cards">
+    <div class="margin-top">
+    </div>
+      <!-- 
+      <div id="simpleModal" class="modal">
+      
+        
+        <div class="work-content-modal">
+          
+          <span class="closeBtn">&times;</span>
+
+          <h2 class="tonic0-modal">Tonic</h2>
+          <ul class="prop-modal">
+            <li class="can">CANOPY</li>
+            <li class="bac"><img src="images/Counter.png" alt="dot"> Back End Dev</li>
+            <li class="num"><img src="images/Counter.png" alt="dot"> 2015</li>
+          </ul>
+      
+          <div class="portfolio-modal-img">
+            <img src="images/Snapshoot-Portfolio.svg" alt="card-1-image">
+          </div>
+      
+          <p class="tonic-def-modal">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum
+            has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into
+            electronic typesetting, remaining essent</p>
+          <ul class="stack stack-desktop">
+            <li class="html">html</li>
+            <li class="css">css</li>
+            <li class="js">javaScript</li>
+          </ul>
+
+          <img src="images/Separator-Bottom.png" alt="separator">
+
```

Comment:
- [ ] Kindly remove all the non descriptive commented code in this file for a cleaner codebase